core/src/main/java/org/lflang/diagram/synthesis/styles/LinguaFrancaShapeExtensions.java

```diff
@@ -839,53 +879,6 @@ public KText addTextButton(KContainerRendering container, String text) {
     return textToAdd;
   }
 
-  /** Creates the triangular line decorator with text. */
-  public KPolygon addActionDecorator(KPolyline line, String text) {
```

Comment:
I suppose this might no longer be necessary? If this is the case, please resolve this comment.

Reply:
This was not called anywhere, so I deleted it.